src/components/app/details/appDetails/DeploymentStatusModal.tsx

```diff
@@ -0,0 +1,146 @@
+                        {lastDeploymentStatus}
+                    </p>
+                </div>
+                <button type="button" className="transparent flex icon-dim-24" onClick={close}>
+                    <Close className="icon-dim-24" />
+                </button>
+            </div>
+            <div className="p-20">
+                <div className="position-rel add-line add-line--git-push">
+                    <div className="flex left ">
+                        <div className="fs-14 cn-9">1/4</div>
+                        <div className="mr-18 ml-17 icon-dim-20 bcn-0 position-rel">
+                            <AppDeploymentStageStatusIcon status={gitPushStep} />
+                        </div>
+                        <div className="pt-12 pb-12" >
+                            <div className={(gitPushStep === "error") ? " lh-1-5 fs-14 cr-5" : gitPushStep === "in_progress" ? " lh-1-5 fs-14 cn-9 fw-`6" : (gitPushStep === "success") ? "lh-1-5 fs-14 cn-9 " : "lh-1-5 fs-14 o-5"}>
+                                {gitPushStep === "error" ? `${message.error}: ` : null}Push to git
+                            </div>
+                            <div className="cn-7">
+                                {gitPushStep === "error" ? <div >Error in pushing configuration to git.</div> : ""}
+                                {gitPushStep === "in_progress" ? <div >Pushing configuration to git.</div> : ""}
+                                {gitPushStep === "success" ? <div >Configuration pushed to git.</div> : ""}
+                                {!(gitPushStep === "success" || gitPushStep === "in_progress" || gitPushStep === "error") ? <div className="o-5">Configuration will be pushed to git.</div> : null}
+                            </div>
+                        </div>
+                    </div>
+                    {gitPushStep === "error" ? <div className="flex left ml-38">
+                        <div className="w-100 cr-5 bcr-1 ml-24 p-12 br-4"><div className="cn-9 fw-6">Error Message:</div>
+                            {deploymentStatus?.gitPushStep?.errorMessage}</div>
+                    </div> : null}
+                </div>
+                <div className="position-rel add-line">
+                    <div className="flex left ">
+                        <div className="fs-14 cn-9">2/4</div>
+                        <div className="mr-18 ml-17 icon-dim-20 bcn-0 position-rel">
+                            <AppDeploymentStageStatusIcon status={gitPullStep} />
+                        </div>
+                        <div className="pt-12 pb-12" >
+                            <div className={(gitPullStep === "error") ? "fs-14 cr-5" : (gitPullStep === "in_progress") ? "fs-14 cn-9 fw-6" : (gitPullStep === "success") ? "fs-14 cn-9 " : "fs-14 o-5"}>
+                                {gitPullStep === "error" ? `${message.error}: ` : null}Pull from git
```

Comment:
should it not be empty string instead of null?

Reply:
it should be empty string. I have updated